recipes/repository.rb

```diff
@@ -49,8 +49,15 @@
 when 'redhat', 'centos'
   # CentOS 6 and 7 have Gluster in the Storage SIG instead of a gluster hosted repo
   if node['platform_version'].to_i > 5
-    repo_url = "http://mirror.centos.org/centos/$releasever/storage/$basearch/gluster-#{node['gluster']['version']}/"
-    gpg_url = 'https://raw.githubusercontent.com/CentOS-Storage-SIG/centos-release-storage-common/master/RPM-GPG-KEY-CentOS-SIG-Storage'
+    if Chef::VersionConstraint.new('>= 3.9').include?(node['gluster']['version'])
+      subdomain = 'buildlogs'
```

Comment:
3.9 was released several weeks ago so we shall see :) 
http://www.gluster.org/pipermail/gluster-users/2016-November/029281.html